pkgs/numpy/numpy.yaml

```diff
@@ -30,6 +30,29 @@ build_stages:
       export LAPACK=None
       export BLAS=None
 
+  - when: machine == 'SGIICEX'
+    name: create-site.cfg
+    after: prologue
+    before: install
+    handler: bash
+    bash: |
+      cat > site.cfg << EOF
+      [mkl]
+      library_dirs = /app/intel/compilers/14/mkl/lib/intel64
```

Comment:
This is about linking Lapack and Blas. We've discussed this at length in some other issue, I think the way forward is to export Hashstack variables about where to find the given blas, and then in the user profile enable it. It's sort of like that already, but it's not finished. Then from here, i.e. form `numpy.yaml`, you just use something like `library_dirs=${LAPACK_DIRS}` or something like that.